src/main/java/seedu/address/model/ModelManager.java

```diff
@@ -210,6 +216,11 @@ public void commitAddressBook() {
                 throw new ItemNotFoundException();
             }
             selectedPerson.setValue((Person) item);
+        } else if (clazz == Booking.class) {
+            if (item != null && !filteredBookings.contains(item)) {
```

Comment:
Should probably put the null check in front- throw exception immediately if the item is null